x11-packages/inkscape/build.sh

```diff
@@ -16,8 +17,34 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 "
 
 termux_step_pre_configure() {
```

Comment:
Is it possible to build graphicsmagick statically in `termux_step_post_make_install` step of graphicsmagick? It is a little strange to build it here.

Reply:
Probably it will be better to recompile it with `-fPIC` and not build PIC version for static linking separately. But I just do not know if it will have side effects or not.

Reply:
@truboxl I need your opinion.

Reply:
`-fPIC` is enabled in clang by default, unless passing `-fno-pic` related flags to `clang`. Recompiling it with `-fPIC` should be safe, as it should be a no-op.